### DISCUSSION TITLE
Opening Keynote [AI2849K] Playback

### DISCUSSION THREAD

**Discussion by msdz** [community/community#74758] [Other Feature Feedback, Questions, & Ideas]
[Click to view on Session catalog](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1695657268465001jL1D
)
[Link to Youtube (temp)](https://youtu.be/NrQkdDVupQE)

**Comment by eminemahjoub** [comment]
thanks <3

**Reply by gitstua** [reply]
Thanks for sharing @msdz! Hope you are enjoying GitHub Universe 2023 as much as we are!!!

 I am going to close off this thread now.
